docs/sections/user_guide/installation.rst

```diff
@@ -18,7 +18,7 @@ Using a fresh Miniforge installation
 
```

Comment:
On line 17, it says

> installed in **the your** installation’s base environment

Reply:
Fixed (docs will rebuild shortly), thank you.